language/Syntax.md

```diff
@@ -97,6 +97,13 @@ multiply x y = x * y
 
 Type signatures are not required for top-level declarations in general, but is good practice to do so. See the section on types for more details.
 
+## Function names
```

Comment:
We should call this section "value names", since these rules apply to all values, not just functions. Eg

```
-- not ok, starts with a number
10 = 10

-- ok
_10 = 10
```

Reply:
I thought values were just "functions of zero arguments".

Reply:
I mean, you can think of them like that if you want to, but it’s not that helpful a perspective in my view.